deps/wxWidgets/wxWidgets.cmake

```diff
@@ -26,6 +26,7 @@ orcaslicer_add_cmake_project(
     wxWidgets
     GIT_REPOSITORY "https://github.com/SoftFever/Orca-deps-wxWidgets"
     GIT_SHALLOW ON
+    GIT_TAG a675dbe8fd07cb5c982cdab7b93cffc9d9f05f22
```

Comment:
Do we need to specify this since the a675dbe8fd07cb5c982cdab7b93cffc9d9f05f22 is the latest commit in master branch?